docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,387 @@
+
+We've also added a **Public dashboard users** tab in **Administration > Users** where you can view a list of users who have accessed your public dashboards by way of a email sharing.
+
+To try it out, please contact customer support.
+
+This feature will have a cost by active users after being promoted into general availability.
+
+To learn more, check out our public dashboards [documentation]({{< relref "../../dashboards/dashboard-public" >}}).
+
+## Authentication and authorization
+
+### Configure your SAML provider in the Grafana UI
+
+<!-- Vardan Torosyan -->
+
+_Generally available in Grafana Enterprise, Cloud Pro, and Cloud Advanced._
+
+You can now configure SAML using our new user interface, making the process easier and more convenient than ever before.
+With the new user interface, you can now configure SAML without needing to restart Grafana and you can control access to the configuration UI by using [role-based access control (RBAC)]({{< relref "../administration/roles-and-permissions/access-control/" >}}), which makes the process much faster and more efficient.
+
+The SAML UI is available in Grafana Enterprise and Grafana Cloud Pro and Advanced. It is intuitive and user-friendly, with clear instructions and helpful prompts to guide you through the process.
+
+For more information on how to set up SAML using the Grafana UI, refer to [Configure SAML authentication using the Grafana user interface]({{< relref "../setup-grafana/configure-security/configure-authentication/saml-ui/" >}}).
+
+### Case-insensitive usernames and email addresses
+
+<!-- Vardan Torosyan -->
+
+_Generally available in all editions of Grafana._
+
+Usernames and email addresses are now treated as case-insensitive, which means that you will no longer need to worry about capitalization when logging in or creating an account.
+
+From now on, whether you type your username or email address in uppercase, lowercase, or a combination of both, Grafana will treat them as the same. This will simplify the login process, reduce the risk of typos and identity conflicts when changing authentication providers.
+
+To help you with dealing with potential user identity conflicts, we have built a [Grafana CLI user identity conflict resolver tool](https://grafana.com/blog/2022/12/12/guide-to-using-the-new-grafana-cli-user-identity-conflict-tool-in-grafana-9.3/) which is available since Grafana 9.3.
+
+Note that if you are running Grafana with MySQL as a database, this change does not have any impact as MySQL users were already treated as case-insensitive.
+
+## Tracing
+
+### Span filtering for traces
+
+<!-- Timur Olzhabayev -->
+
+_Available in preview in all editions of Grafana._
```

Comment:
For that phrase, I'm not entirely sure if it's correct. The [wording here](https://grafana.com/docs/release-life-cycle/) would suggest it is and it isn't, so not sure should this be updated. It's only going to be toggled on by default for hosted Grafana customers, using [release waves](https://github.com/grafana/deployment_tools/tree/master/ksonnet/environments/hosted-grafana/waves/feature-toggles#instructions) i.e. enabling by default in the canary and prod waves. This is a relatively new feature so it's not GA just yet but we would like to make it easier for customers to give us feedback.

Reply:
@joey-grafana  I went through some of the other docs and found this phrasing for preview features. What do you think? Is this a public preview or a private preview? I'm guessing public because the feature will be available for hosted Grafana. I updated the wording slightly for public preview. What do you think? 

The paragraph where you mention the feature flag also talks about availability for the feature. Does the combination of that paragraph and the suggested wording below cover things?
```suggestion
_Span filtering for traces is currently in public preview. Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available._
```

Reply:
Hi Kim, thank you. The wording here makes sense although I would imagine we would not introduce any breaking changes - more like the UI design may change. It sounds more like a public preview than a private one. I would say that you can make the call on the final wording we use together with whomever is in charge of reviewing this PR. This new feature will be available in Grafana Cloud only for the time being.